packages/components/src/components/post-menu/post-menu.tsx

```diff
@@ -81,10 +93,11 @@ export class PostMenu {
    */
   @Method()
   async toggle(target: HTMLElement) {
-    if (this.isVisible) {
-      await this.hide();
+
+    if (this.popoverRef) {
+      await this.popoverRef.toggle(target);
```

Comment:
Why is the keyword `await` needed here? You're not doing anything else afterwards.